torax/_src/geometry/imas.py

```diff
@@ -177,14 +179,17 @@ def geometry_from_IMAS(
       "z_magnetic_axis": z_magnetic_axis,
   }
 
-
+#TODO: For future PR, if imas files get refactored, modify the function to be
```

Comment:
Can this be refactored now into a function which takes a full uri and ids_name and then one which takes the geo directory, concatenates using the path join and then calls this